src/coreclr/src/gc/bitonic_sort.int64_t.generated.h

```diff
@@ -0,0 +1,1481 @@
+
```

Comment:
Comment about how this file was generated would be nice

Reply:
The script is part of my repo:

https://github.com/damageboy/gcsort/blob/master/gcsort/smallsort/bitonic_gen.py

Note that it generates all 6 32+64 bit types.

Reply:
By the way, I would highly recommend including the generator and reviewing it instead of the mountains of code it generated.

It might actually make sense that way :)

Reply:
@jkotas I've updated the [original repo](https://github.com/damageboy/gcsort/blob/master/gcsort/smallsort/bitonic_sort.int64_t.generated.h) to now include proper auto-generation disclaimers:

```c
/////////////////////////////////////////////////////////////////////////////
////
// This file was auto-generated by a tool at 2020-05-30 11:18:44
//
// It is recommended you DO NOT directly edit this file but instead edit
// the code-generator that generated this source file instead.
/////////////////////////////////////////////////////////////////////////////
```

I'm assuming we'll do another pass of updating the gc copy from my original given there will be more changes stemming from this review process.

Reply:
we'd definitely want to include the generator. 
BTW @PeterSolMS is OOF, that's why he hasn't said anything, he will be back this coming Tue.

Reply:
I wonder whether to put the generator under src/tools or src/gc?

Reply:
We typically place generators like this locally to the thing they are generating. Here are a few existing examples:
https://github.com/dotnet/runtime/blob/master/src/coreclr/src/inc/CrstTypeTool.cs
https://github.com/dotnet/runtime/tree/master/src/coreclr/src/System.Private.CoreLib/Tools/GenUnicodeProp